libs/pika/async_cuda/tests/performance/CMakeLists.txt

```diff
@@ -4,7 +4,12 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-set(benchmarks cuda_scheduler_throughput synchronize)
+set(benchmarks cuda_scheduler_throughput)
+
+if(NOT PIKA_WITH_CLANG_CUDA)
+  list(APPEND benchmarks synchronize)
```

Comment:
This configuration is stranger than I hoped. It times out _building_ `tests.performance.modules.async_cuda.synchronize`. Corresponding issue: https://github.com/pika-org/pika/issues/456.